patches/debian_buster/fixes/mojo.patch

```diff
@@ -3,23 +3,12 @@ author: Michael Gilbert <mgilbert@debian.org>
 
 --- a/content/shell/BUILD.gn
 +++ b/content/shell/BUILD.gn
-@@ -304,10 +304,7 @@ static_library("content_shell_lib") {
+@@ -435,8 +435,6 @@ static_library("content_shell_lib") {
+     "//content/public/common",
      "//content/public/common:service_names",
-     "//content/shell/test_runner:test_runner",
      "//content/test:blink_test_browser_support",
 -    "//content/test:content_test_mojo_bindings",
 -    "//content/test:mojo_web_test_bindings",
      "//content/test:test_support",
--    "//content/test:web_test_support",
+     "//content/test:web_test_support_browser",
```

Comment:
`web_test_support_browser` appears to contain a subset of the code in the old `web_test_support`. Are you confident that leaving it in fulfills the purpose of this patch?

Reply:
Apparently the [purpose of this patch](https://sources.debian.org/patches/chromium/83.0.4103.116-1%7Edeb10u3/fixes/mojo.patch/#L1) is to:
> fix mojo layout test build error

My sequence of thoughts after your comment:
1. Build does not trow this error (maybe only if test are run?)
1. But, the current version of [this patch on sources.debian.org](https://sources.debian.org/patches/chromium/83.0.4103.116-1%7Edeb10u3/fixes/mojo.patch/) does not remove that test anymore
1. Why is this patch in the macos repo anyway?
  1. However, I did not try to build without it
1. The series of patches should probably be checked regarding which of them are still needed/up-to-date
  1. I'm not going to have the resources to do that - at least in the short term

Suggestion: I can remove the `//content/test:web_test_support` line if you think it's better to do so, but I think it probably makes no difference

Reply:
I think you are correct. The fact that the upstream version of the patch no longer removes `web_test_support` means that leaving `web_test_support_browser` should be fine.